lib/src/widgets/dashboard_screen/summary.dart

```diff
@@ -23,132 +23,212 @@ class Summary extends StatelessWidget {
                           children: <Widget>[
                             Text(
-                              'See card list',
+                              'SEE ALL ' +
+                                  totalCardsNumber.toString() +
+                                  ' CARDS',
                               overflow: TextOverflow.ellipsis,
-                              style: SarakaTextStyles.headlineSmall,
-                              textAlign: TextAlign.center,
+                              style: SarakaTextStyles.heading.copyWith(
+                                  fontSize: 20, color: SarakaColors.darkGray),
                             ),
                             Icon(
-                              Icons.keyboard_arrow_down,
+                              Icons.arrow_forward_ios,
                               color: SarakaColors.darkGray,
+                              size: 20,
                             ),
                           ],
                         ),
-                        onTap: () => Navigator.of(context).pushNamed('/cards'),
                       ),
+                      onPressed: () =>
+                          Navigator.of(context).pushNamed('/cards'),
                     ),
                   ],
-                )),
-              ],
-            ),
-          ),
-          Expanded(
-            child: Column(
-              mainAxisAlignment: MainAxisAlignment.end,
-              children: <Widget>[
-                Text(
-                  todayLearnNumber + ' cards you study for today',
-                  overflow: TextOverflow.ellipsis,
-                  style: SarakaTextStyles.headlineSmall,
-                  textAlign: TextAlign.center,
                 ),
                 Padding(
-                  padding: EdgeInsets.only(bottom: 88.0),
-                )
+                  padding: EdgeInsets.only(bottom: 32),
+                ),
+                Column(
+                  children: <Widget>[
+                    Text(
+                      '  cards will be mature',
+                      overflow: TextOverflow.ellipsis,
+                      style: SarakaTextStyles.body,
+                    ),
+                    Row(
+                      mainAxisAlignment: MainAxisAlignment.center,
+                      children: <Widget>[
+                        Padding(
+                          padding: EdgeInsets.only(right: 48),
+                        ),
+                        Text(
+                          'within   days to come',
+                          overflow: TextOverflow.ellipsis,
+                          style: SarakaTextStyles.body,
+                        ),
+                      ],
+                    ),
+                  ],
+                ),
               ],
             ),
           ),
+          Container(
+            padding: EdgeInsets.only(bottom: 80),
+            child: Text(
+              todayLearnNumber + ' cards you study for today',
```

Comment:
Sounds kinda strange as English. I prefer `X cards to study today` or `X cards you will study today`

Reply:
Done.